xml/deployment_cifs.xml

```diff
@@ -0,0 +1,290 @@
+       Make sure the following packages are installed on &si.earth; and &si.mars;
+       before you proceed: <package>ctdb</package>,
+       <package>tdb-tools</package>, and <package>samba</package>
+       (needed for smb and nmb resources). 
+      </para>
+<screen>&prompt.root;<command>zypper</command> in ctdb tdb-tools samba</screen>
+     </step>
+     <step>
+      <para>
+       Make sure the services <literal>ctdb</literal>, <literal>smb</literal>,
+       and <literal>nmb</literal> are disabled: 
+      </para>
+<screen>&prompt.root;<command>systemctl</command> disable ctdb
+&prompt.root;<command>systemctl</command> disable smb
+&prompt.root;<command>systemctl</command> disable nmb</screen>
+     </step>
+     <step>
+      <para>
+       Open port <literal>4379</literal> of your firewall on all nodes. This
+       is needed for CTDB to communicate with other cluster nodes.
+      </para>
+     </step>
+    </substeps>
+   </step>
+   <step>
+    <para>
+     Create a directory for the CTDB lock on the shared file system:
+    </para>
+<screen>&prompt.root;<command>mkdir</command> -p /srv/samba/</screen>
+   </step>
+   <step>
+    <para>
+     In <filename>/etc/ctdb/nodes</filename> insert all nodes which contain
+     all private IP addresses of each node in the cluster:
+    </para>
+<screen>&subnetI;.1
+&subnetI;.2</screen>
+   </step>
+   <step>
+    <para>
+     Configure Samba. Add the following lines in the
+     <literal>[global]</literal> section of
+     <filename>/etc/samba/smb.conf</filename> on &si.earth; and &si.mars;.
+     Use the host name of your choice in place of "CTDB-SERVER" (all nodes
+     in the cluster will appear as one big node with this name, effectively):
+    </para>
+<screen>[global]
+    # ...
+    # settings applicable for all CTDB deployments
+    netbios name = CTDB-SERVER
+    clustering = yes
+    idmap config * : backend = tdb2
+    passdb backend = tdbsam
+    ctdbd socket = /var/lib/ctdb/ctdb.socket
+    # settings necessary for CTDB on OCFS2
+    fileid:algorithm = fsid
+    vfs objects = fileid
+    # ...</screen>
+    <para>
+     Alternatively, <command>csync2</command> can be used to synchronize
+     config files. For details, see
+     <link xlink:href=
+     "https://www.suse.com/documentation/sle-ha-12/singlehtml/book_sleha/book_sleha.html#pro.ha.installation.setup.csync2.start"
+     />.
+    </para>
+   </step>
+   <step>
+    <para>
+     Register the &sle; &hasi; on &si.earth; and &si.mars;.
+    </para>
+    <screen>
+&prompt.root;<command>SUSEConnect</command> -r <replaceable>ACTIVATION_CODE</replaceable> -e <replaceable>E_MAIL</replaceable>
+    </screen>
+   </step>
+   <step>
+    <para>
+     Install <package>ha-cluster-bootstrap</package> on both nodes:
+    </para>
+    <screen>
+&prompt.root;<command>zypper</command> in ha-cluster-bootstrap
+    </screen>
+   </step>
+   <step>
+    <substeps>
+     <step>
+      <para>
+       Init the cluster on &si.earth;:
```

Comment:
"Init" is jargon. 😉 Better rephrase it like "Create a one-node cluster on &si.earth;:"